src/patternfly/components/Drawer/drawer-section.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-drawer__section{{#if drawer-section--modifier}} {{drawer-section--modifier}}{{/if}}"
```

Comment:
Whats the purpose of drawer-section here above the content?

<img width="479" alt="Screen Shot 2020-02-10 at 4 04 31 PM" src="https://user-images.githubusercontent.com/20118816/74189827-16e49e00-4c1f-11ea-9dda-26bed738a6c7.png">

Reply:
@christiemolloy @mcoker and I discussed this at length. He asked the same question previously in the comments too. Basically, it includes an element (perhaps a toolbar) that is not affected by the `__panel`, but part of the drawer. You may ask, can you just stack them? The answer is no because the drawer spec states that `drawer` must be 100% height of its parent, which makes sense. 

So stacking results in this because `drawer`s height is 100% (which it should be):
![Screen Shot 2020-02-07 at 2 52 32 PM](https://user-images.githubusercontent.com/5385435/74353812-73af9800-4d88-11ea-85a0-ad26481a7ff5.png)

Removing `height: 100%` results in this:
![Screen Shot 2020-02-07 at 2 52 19 PM](https://user-images.githubusercontent.com/5385435/74353814-74482e80-4d88-11ea-8035-b99aab62750c.png)

You may also ask, why we wouldn't use a layout to set height. This approach seems pretty inefficient and impractical to expect our devs to know to do use a layout. It would also lead to breakages and visual inconsistencies.

Reply:
It's part of the master/detail layout concept. [This example](https://patternfly-next-pr-2645.surge.sh/documentation/core/demos/masterdetail/master-detail-card-view-expanded) uses it.

https://marvelapp.com/9iefh5b/screen/63567619

Reply:
AH got it!! Okay thank you so much for explaining that!!!